packages/toolpad-components/src/Form.tsx

```diff
@@ -263,7 +260,12 @@ export function useFormInput<V>({
   React.useEffect(() => {
     if (
       form &&
-      !isEqual(validationProps, previousManualValidationPropsRef.current) &&
+      !(
+        validationProps.isInvalid === previousManualValidationPropsRef.current.isInvalid &&
```

Comment:
I'm not sure why I called this `previousManualValidationPropsRef` if I did, I guess it could be just `previousValidationPropsRef`?